cpp/merkletree/compact_merkle_tree.cc

```diff
@@ -21,7 +21,7 @@ CompactMerkleTree::CompactMerkleTree(SerialHasher* hasher)
 
 CompactMerkleTree::CompactMerkleTree(MerkleTree& model, SerialHasher* hasher)
     : MerkleTreeInterface(),
-      tree_(std::max(0L, int64_t(model.LevelCount()) - 1)),
+      tree_(std::max(int64_t(0), int64_t(model.LevelCount()) - 1)),
```

Comment:
I'd like that better, because C-style casts make me hiss, due to the occasional silent promotion to `reinterpret_cast` (where the C++ ones are a bit less dangerous): http://en.cppreference.com/w/cpp/language/explicit_cast